### DISCUSSION TITLE
Novos prazos para movimentações de reembolsos

### DISCUSSION THREAD

**Discussion by tiidadavena** [apyb/comunidade#143] [Geral]
# Novos prazos para movimentações de reembolsos 

<img src="https://github.com/user-attachments/assets/336200f4-f66b-4182-88fe-a870c80409bd" alt="Imagem com título Novo prazo para solicitação de reembolsos e com a logo da Associação Python Brasil embaixo" width="300px"/>

---

Olá, pythonistas!

Você organizou eventos usando a estrutura da APyB? Saiba que temos novidades sobre os reembolsos.

Como foi publicado no Boletim Informativo de abril deste ano, seguem novas regras:

1- Todos os reembolsos devem ser solicitados no máximo até 30 dias após o gasto realizado;

2- O período máximo para solicitação de pagamento ou reembolso será 60 dias após a realização do evento;

Os prazos servem para otimizar a atuação da Diretoria Financeira no tocante ao recebimento dos valores, fechamento mensal da APyB. O objetivo final é prover uma movimentação financeira e fluida para as pessoas associadas.

🔗  O link para submissão de reembolso pode ser encontrado [aqui](https://www.ultradox.com/app/apyb-solicitacao-reembolso).

> Dúvidas? Envie-uns um email para: [contato@python.org.br](mailto:contato@python.org.br)

**Comment by teodororo** [comment]
Olá, bom dia. Uma dúvida: a APyB tem prazo para **resposta** sobre o reembolso? Solicitei o do evento `pydatamanaus7` há alguns dias e não obtive resposta via e-mail. Preciso fechar o documento de prestação de contas... :crying_cat_face:
